packages/filestream/manifest.yml

```
@@ -29,15 +29,15 @@ policy_templates:
         show_user: true
         default:
           - /var/log/*.log
-      - name: gzip_experimental
+      - name: compression_gzip
         type: bool
         title: Decompress GZIP files
         multi: false
         required: false
         show_user: true
         default: false
         description: |
-          When enabled, filestream will decompress GZIP-compressed files (.gz) as they are read. This is beta feature. For full details, see the [documentation](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-filestream#reading-gzip-files). If enabled, you **must** also remove the '\.gz$' pattern from the "Exclude Files" setting to ensure `.gz` files are ingested. Available for Elastic Agent 9.2.0 and newer.
+          When enabled, filestream will decompress GZIP-compressed files (.gz) as they are read. For full details, see the [documentation](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-filestream#reading-gzip-files). If enabled, you **must** also remove the '\.gz$' pattern from the "Exclude Files" setting to ensure `.gz` files are ingested. Available for Elastic Agent 9.2.0 in beta and for and Elastic Agent 9.3.0 or newer in GA.
```

The link for the documentation is not working, I believe it should be https://www.elastic.co/docs/reference/beats/filebeat/filebeat-input-filestream#reading-gzip-files:
```suggestion
          When enabled, filestream will decompress GZIP-compressed files (.gz) as they are read. For full details, see the [documentation](https://www.elastic.co/docs/reference/beats/filebeat/filebeat-input-filestream#reading-gzip-files). If enabled, you **must** also remove the '\.gz$' pattern from the "Exclude Files" setting to ensure `.gz` files are ingested. Available for Elastic Agent 9.2.0 in beta and for and Elastic Agent 9.3.0 or newer in GA.
```